frontend/src/app/main/ui/dashboard/team.cljs

```diff
@@ -571,6 +577,229 @@
+     [:h2 (tr "labels.webhooks")]
+     [:& i18n/tr-html {:label "dashboard.webhooks.description"}]]
+
+    [:div.btn-primary
+     {:on-click #(st/emit! (modal/show :webhook {}))}
+     [:span (tr "dashboard.webhooks.create")]]]])
+
+
+  (mf/defc webhook-actions
+    [{:keys [on-edit on-delete] :as props}]
+    (let [show? (mf/use-state false)]
+      [:*
+       [:span.icon {:on-click #(reset! show? true)} [i/actions]]
+       [:& dropdown {:show @show?
+                     :on-close #(reset! show? false)}
+        [:ul.dropdown.actions-dropdown
+         [:li {:on-click on-edit} (tr "labels.edit")]
+         [:li {:on-click on-delete} (tr "labels.delete")]]]]))
+
+  (mf/defc last-delivery-icon
+    [{:keys [success? text] :as props}]
+    [:div.last-delivery-icon
+     [:div.tooltip
+      [:div.label text]
+      [:div.arrow-down]]
+     (if success?
+       [:span.icon.success i/msg-success]
+       [:span.icon.failure i/msg-warning])])
+
+  (mf/defc webhook-item
+    {::mf/wrap [mf/memo]}
+    [{:keys [webhook] :as props}]
+    (let [on-edit #(st/emit! (modal/show :webhook {:webhook webhook}))
+          error-code (:error-code webhook)
+          extract-status
+          (fn [error-code]
+            (let [status (-> error-code
+                             (str/split  "-")
+                             last)]
+              (if (js/isNaN status)
+                ""
+                (js/parseInt status))))
+          delete-fn
+          (fn []
+            (let [params {:id (:id webhook)}
+                  mdata  {:on-success #(st/emit! (dd/fetch-team-webhooks))}]
+              (st/emit! (dd/delete-team-webhook (with-meta params mdata)))))
+          on-delete #(st/emit! (modal/show
+                                {:type :confirm
+                                 :title (tr "modals.delete-webhook.title")
+                                 :message (tr "modals.delete-webhook.message")
+                                 :accept-label (tr "modals.delete-webhook.accept")
+                                 :on-accept delete-fn}))]
+      [:div.table-row
+       [:div.table-field.last-delivery
+        [:div.icon-container (if (nil? error-code)
+                               [:& last-delivery-icon {:success? true :text (tr "dashboard.webhooks.last-delivery.success")}]
+                               [:& last-delivery-icon {:success? false :text
+                                                       (cond
+                                                         (= error-code "ssl-validation") (tr "dashboard.webhooks.last-delivery.error.ssl-validation")
+                                                         (str/starts-with? error-code "unexpected-status")
+                                                         (tr "dashboard.webhooks.last-delivery.error.unexpected-status" (extract-status error-code))
+                                                         :else (tr "dashboard.webhooks.last-delivery.error.unknown"))}])]]
```

Comment:
I guess is better to put the error handling on the let and here just put the string on the `:text`
Also, about the translation strings, I think we don't need to add this nesting:
`dashboard.webhooks.last-delivery.error.ssl-validation`

I guess all errors can be look like `errors.xxxx.yyyy`
As webhooks is a general concept, that is not related directly to dashboard (only the current location on the UI) so I suggest using `errors.webhooks.ssl-validation`
excesive nesting tends to reduce reusability and more duplication
Also, if have a sufficiently generic term, just put it under `labels.generic-term`

Reply:
Fixed